packages/components/src/SubHeaderBar/TitleSubHeader/TitleSubHeader.component.js

```diff
@@ -109,15 +109,22 @@ function SubTitle({ subTitleLoading, subTitle }) {
 	}
 
 	if (subTitle) {
+		let subtitleClassnames;
+		if (asLabel) {
+			subtitleClassnames = classNames('label', 'tc-subheader-details-text-subtitle-label', {
+				'label-info': labelType === 'INFO',
+			});
+		} else {
+			subtitleClassnames = classNames(
+				theme['tc-subheader-details-text-subtitle'],
+				'tc-subheader-details-text-subtitle',
+			);
+		}
+
 		return (
-			<small
-				className={classNames(
-					theme['tc-subheader-details-text-subtitle'],
-					'tc-subheader-details-text-subtitle',
-				)}
-			>
-				{subTitle}
-			</small>
+			<div>
+				<small className={subtitleClassnames}>{subTitle}</small>
```

Comment:
Why don't you use `Label` from `react-bootstrap` as it is done in `CollapsiblePanel`, for example ?

![image](https://user-images.githubusercontent.com/26482371/74352374-af625600-4db8-11ea-85ed-e7c3a520c337.png)